drivers/UnbufferedSerial.h

```diff
@@ -160,6 +160,9 @@ class UnbufferedSerial:
     using SerialBase::format;
     using SerialBase::attach;
     using SerialBase::baud;
+    using SerialBase::RxIrq;
+    using SerialBase::TxIrq;
+    using SerialBase::IrqCnt;
```

Comment:
Why do you need to expose SerialBase::IrqCnt?

Reply:
I just followed the serial flow control enums has shared, It helps on validating the enum
```   
    using SerialBase::Disabled;
    using SerialBase::RTS;
    using SerialBase::CTS;
    using SerialBase::RTSCTS;